StockTrader/src/main/resources/schema.sql

```diff
@@ -37,45 +37,48 @@ CREATE TABLE account (
 );
 
 CREATE TABLE account_transaction_log (
-    action varchar,
+    transaction_num SERIAL PRIMARY KEY,
+    action varchar(255),
     funds integer,
-    timestamp timestamp,
-    username varchar
+    timestamp bigint,
```

Comment:
why is this changed to bigint?

Reply:
Before it was timestamp, but we need to log the timestamp in xml output as unix time milliseconds. The xml schema that she checks against checks to see if the timestamp values are between a range of unix milliseconds. This was just a way to get the milliseconds into the log instead of a timestamp mont-day-year type of value.